### PR TITLE
Prepping for 1.2.4 release

### PR DESCRIPTION
**Pull request by jlaura**

### CODE REVIEW

**Comment by ladoramkershner** [comment]
Just waiting for the tests to pass

File: appveyor.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@ environment:
       PYTHON_VERSION: 3.6
     - PYTHON: "C:\\Miniconda37-x64\\Scripts\\activate.bat"
       PYTHON_VERSION: 3.7
-
+    - PYTHON: "C:\\Miniconda38-x64\\Scripts\\activate.bat"
```
Comment (jessemapel) [inline appveyor.yml:16]: Strange that appveyor can't find this it's listed on their docs https://www.appveyor.com/docs/windows-images-software/#miniconda

**Comment by jlaura** [comment]
I also do not understand what is going on with appveyor. I saw the same docs, so assumed this should be working...Reran the job once to make sure this wasn't just an intermittent thing.

 I can roll back 3.8 support on win or this can get shelved for a bit until I have more time. Thoughts?

**Comment by ladoramkershner** [comment]
Is this release just to get out the most recent plio change (PR #148)?

**Comment by jlaura** [comment]
@ladoramkershner Yup. Just to get the release out. So, if it sits for a bit it isn't a big deal as along as you are good rolling. Typing that, maybe the best thing to do is for me to open an issue and get to that in the future.

**Comment by jlaura** [comment]
@ladoramkershner I'm doing a test now. Based on how the file is structured, it looks like we can specify 3-x64 to get miniconda3 installed. Then we set up an environment with the PYTHON_VERSION version installed, so any 3 should work. I'm testing on the 3.8.

**Comment by jlaura** [comment]
@ladoramkershner @jessemapel This was fixed using the approach above. Here is the line in the 3.8 where the correct python is installed: https://ci.appveyor.com/project/jlaura/plio/build/job/uutxlso6tvtx8tu0#L118

Tests are using 3.8 like we want: https://ci.appveyor.com/project/jlaura/plio/build/job/uutxlso6tvtx8tu0#L882

I won't know about the Conda build and release until this is merged because we don't conda build a PR.

**Pull request merged**